app/models/setting/provisioning.rb

```diff
@@ -3,12 +3,16 @@ class Setting::Provisioning < Setting
   def self.load_defaults
     # Check the table exists
     return unless super
-
+    blank = { nil => _("select an owner")}
+    users = Hash[User.visible.map{|usr| [usr.id_and_type, usr[:login]]}]
```

Comment:
I think you need to scope this by organization/location. Currently I can set a `default_owner` that is not in my current organization/location.